numpy/core/src/multiarray/methods.c

```diff
@@ -2110,6 +2111,8 @@ PyArray_Dump(PyObject *self, PyObject *file, int protocol)
     if (cpick == NULL) {
         return -1;
     }
+
+    file = NpyPath_PathlikeToFspath(file);
```

Comment:
You need to check for `NULL` here, and return `-1` if you get it.

Reply:
Done

Reply:
You fixed the one above, but not this one.

Reply:
Sorry; missed that one.